contributors/devel/sig-instrumentation/migration-to-structured-logging.md

```diff
@@ -7,6 +7,61 @@ should be applicable at later stages or for other projects using `klog` logging
 
 [Structured Logging KEP]: https://github.com/kubernetes/enhancements/tree/master/keps/sig-instrumentation/1602-structured-logging
 
+## How to contribute
+
+### About the migration
+
+Migration proceeds in two different ways. First is organized milestone based large scale migration where we are trying to achieve a particular target for a Kubernetes release (for example migrate whole Kubelet code in 1.21). Second is not organized migration done by individual contributors.
+
+Organized migration is done in two state cycle aligned with Kubernetes releases. In stage one we pick a particular migration milestone and create issue that splits the work into smaller chunks (for example migrating just single file) that can be easily picked up by contributors. This way we ensure that multiple people can independently work on common goal and avoid conflicts. After migration is done we can utilize tooling we developed to mark particular directory as migrated to avoid regressions. When we finish particular milestone we move the second stage where we take a break from migration, to analyse results and improve on the process. Adding structured information to logs is very costly and time consuming process, we don't want to make a mistake that would require us to repeat the migration. That's why we reserve time to collect feedback, analyse impact of changes we have done on log volume, performance and PR review process.  
+
+Non-organized migration is done by individual contributors that want particular code section to utilize new features early. Migration of Kubernetes components not yet marked for migration is on best effort bases. This means that we don't guarantee a timely review or that PR will be accepted at all. It's up to component owners to decide if they want to accept those contributions. As this migration is not organized it means that two contributors could start working on migrating same code. Before sending a PR please make sure that there is was already created by someone else.
```

Comment:
As aforementioned, our non-organized migration efforts are spearheaded by individual contributors who require particular code sections to utilize new features early. Efforts for Kubernetes components not yet marked for migration also fall under this category & are carried out on a best-efforts basis. This means that we don't guarantee a timely review or that the PR will be accepted at all. The respective component owners have a final say in the acceptance of these contributions. Since this is a non-organized effort there is a high possibility that two contributors could be working on migrating the same code. Before sending a PR our way, please ensure that there isn't one already in place created by someone else.